### ISSUE TITLE
The settings of fine-tune on BlendedMVS

### ISSUE THREAD

**Issue by VILmchen** [QT-Zhu/AA-RMVSNet#12]
Hi! I use the script "train_blend.sh" and the default setting to fine-tune the released pre-trained model on the BlendedMVS and hope to reproduce the result on Tanks and Temples. I have one NVIDIA 3090 (24G), but I met the out-of-memory problem. Do I need to change some settings like the "--max_h" and "--max_w"?

**Comment by QT-Zhu** [comment]
The `train_blend.sh` script has been tested thoroughly in fact... That is why we didn't write this on the main README. But I think 24GB will be enough since the batchsize is 1 per card. Maybe you can try to lower the value of `view_num`?

**Comment by VILmchen** [comment]
OK, I will try to lower the value of view_num. If I use the train_blend.sh script to fine-tune the released pre-trained model, can I obtain the result which closes to the reported result of AA-RMVSNet on the Tanks and Temples? Generally, how many epochs do I need to train to fine-tune the released pre-trained model for the best result?

**Comment by QT-Zhu** [comment]
Yes. Just check the loss when it converges.

**Comment by VILmchen** [comment]
OK, thanks a lot!

**Comment by VILmchen** [comment]
Hello, I use the setting shown in the picture below to fine-tune the released model for testing on Tanks and Temples, but the result is not very well. Is there any wrong with the "train_blend.sh" settings that I have changed? Besides, could you help me to see the training process shown in the tensorboard is normal (the Figure is shown below)?
<img width="416" alt="1" src="https://user-images.githubusercontent.com/49902801/151512737-06e90327-ca8e-4046-a1bf-2ab6798e7c66.png">
<img width="556" alt="2" src="https://user-images.githubusercontent.com/49902801/151512754-6ac60476-f85f-45f4-ba2f-bda288ef5d09.png">
<img width="553" alt="3" src="https://user-images.githubusercontent.com/49902801/151512772-f67f5508-6ec5-4979-9fbe-eb943b6d86a0.png">
<img width="556" alt="4" src="https://user-images.githubusercontent.com/49902801/151512794-b94c718d-68b5-4731-a313-7ffc0396934d.png">
